refactor(curriculum): background and borders quiz questions (#58419)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/25-front-end-development/quiz-css-backgrounds-and-borders/66ed8fd7f45ce3ece4053eb0.md

#### --text--

> What is WCAG's recommended minimum contrast ratio for normal text? 

Author: yeah, I can change it 👍

#### --text--

What is does the `radial-gradient` function do?

#### --distractors--

> It creates a background that radiates and transitions between multiple colors along a straight line.

Author: I wasn't sure if having only the correct answer say radiates would it give it away. So I added it to one of the wrong ones

---

It specifies whether the background image should scroll with the content or remain fixed in place.

---

It is used to style an element that was activated by the user.

#### --answer--

It creates an image that radiates from a particular point and gradually transitions between multiple colors.

### --question--

#### --text--

Which of the following is NOT a valid pseudo-class?

#### --distractors--

`:focus`

---

`:visited`

---

`:link`

#### --answer--

`:before`

### --question--

#### --text--

> Which of the following is the correct way to use the `background` shorthand property?

Reviewer: ```suggestion
Which of the following shows the `background` shorthand property being used properly?
```
Nit: the original implies there's one correct way, but it's just an example of one.
